packages/xs/polly.0.2.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+name: "polly"
+synopsis: "Bindings for the Linux epoll system call"
+description: """
+Bindings for the Linux epoll system call. The binding avoids
+  most allocation in the event loop by iterating over all file
+  descriptios that are reported as ready ."""
+maintainer: "Christian Lindig <christian.lindig@citrix.com>"
+authors: "Christian Lindig <christian.lindig@citrix.com>"
+license: "MIT"
+homepage: "https://github.com/lindig/polly"
```

Comment:
Should we move this inside the xapi project?

Reply:
Or we could leave it there for now, if we intend to merge the repositories it'll end up as a subtree of the merged toolstack repo anyway.

Reply:
Yes - but let's make this a separate step.